discovery/token/README.md

```diff
@@ -1,29 +1,9 @@
 #discovery.hub.docker.com
 
-Docker Swarm comes with a simple discovery service built into the [Docker Hub](http://hub.docker.com)
-
-#####Create a new cluster
-`-> POST https://discovery.hub.docker.com/v1/clusters`
-
-`<- <token>`
-
-#####Add new nodes to a cluster
-`-> POST https://discovery.hub.docker.com/v1/clusters/<token>?ttl=<ttl> Request body: "<ip>:<port1>"`
-
-`<- OK`
-
-`-> POST https://discovery.hub.docker.com/v1/clusters/<token>?ttl=<ttl> Request body: "<ip>:<port2>")`
-
-`<- OK`
-
-
-#####List nodes in a cluster
-`-> GET https://discovery.hub.docker.com/v1/clusters/<token>`
-
-`<- ["<ip>:<port1>", "<ip>:<port2>"]`
-
-
-#####Delete a cluster (all the nodes in a cluster)
-`-> DELETE https://discovery.hub.docker.com/v1/clusters/<token>`
-
-`<- OK`
+Docker Swarm comes with a simple discovery service built into the [Docker
+Hub](http://hub.docker.com).  To read the end-user API documentation, visit [the
+Swarm API documentation on
+docs.docker.com](https://docs.docker.com/swarm/api/swarm-api/). If you want to
+modify the discovery API documentation, start with [the `docs/api/swarm-api.md`
```

Comment:
@moxiegirl I think we can keep a simple README.md in this folder. Something like this: 

```
Docker Swarm comes with a simple discovery service built into the [Docker
Hub](http://hub.docker.com). To read the end-user API documentation, visit [the
Swarm discovery documentation on
docs.docker.com](https://docs.docker.com/swarm/discovery/). If you want to
modify the discovery documentation, start with [the `docs/discovery.md`]
```